packages/frontend/component/src/styles/mui-theme.ts

```diff
@@ -1,86 +0,0 @@
-import type {
-  Breakpoint,
-  BreakpointsOptions,
-  ThemeOptions,
-} from '@mui/material';
-
-export const muiThemes = {
-  breakpoints: {
-    values: {
-      xs: 0,
-      sm: 640,
-      md: 960,
-      lg: 1280,
-      xl: 1920,
-    },
-  },
-} satisfies ThemeOptions;
-
-// Ported from mui
-// See https://github.com/mui/material-ui/blob/eba90da5359ff9c58b02800dfe468dc6c0b95bd2/packages/mui-system/src/createTheme/createBreakpoints.js
-// License under MIT
-function createBreakpoints(breakpoints: BreakpointsOptions): Readonly<
```

Comment:
Since there is currently only one reference and it's based on a function provided by the MUI, it must be removed. 
Responsive encapsulation needs to be considered later.